JhipsterXamarin/JhipsterXamarin/JhipsterXamarin/ViewModels/LoginViewModel.cs

```diff
@@ -10,13 +10,23 @@ namespace JhipsterXamarin.ViewModels
     public class LoginViewModel : MvxViewModel
     {
         private readonly IAuthenticationService _authenticationService;
+
+        public IAuthenticationService GetAuthenticationService()
+        {
+            return _authenticationService;
+        }
         private readonly IMvxNavigationService _navigationService;
 
+        public IMvxNavigationService GetMvxNavigationService()
```

Comment:
_navigationService is also injected with mvvmcross you don't need to make it public nor create a getter in any situations